src/components/Theme.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {ThemeProvider, createGlobalStyle} from 'styled-components'
+import styledNormalize from 'styled-normalize'
+
+const theme = {
+  breakpoints: {
```

Comment:
We start here by setting up a `theme` object. Themes are passed through the context via styled-components' `ThemeProvider`, so now we'll have access to our theme object.

[next ->]()